### PR TITLE
fixup/redis mem

### PR DESCRIPTION
**Pull request by ahuston-0**
Fixes an issue with the autopull service where it starts way too early.

Enables the overcommit sysctl option on palatine-hill, for Redis.

This is staying a draft until I cant test the autopull fix

### CODE REVIEW

**Comment by ahuston-0** [comment]
This should be ready for review. pls take a look

File: systems/palatine-hill/configuration.nix [hunk 1]
```diff
@@ -20,6 +20,10 @@
     filesystem = "zfs";
     useSystemdBoot = true;
     kernelParams = [ "i915.force_probe=56a5" "i915.enable_guc=2" ];
+    kernel.sysctl = {
+      "vm.overcommit_memory" = 1;
+      "vm.swappiness" = 10;
+    };
     binfmt.emulatedSystems = [ "aarch64-linux" ];
```
Comment (DerDennisOP) [inline systems/palatine-hill/configuration.nix:27]:
```suggestion
    binfmt.emulatedSystems = [ "aarch64-linux" ];
    kernel.sysctl = {
      "vm.overcommit_memory" = 1;
      "vm.swappiness" = 10;
    };
```

**Review by DerDennisOP — APPROVED** [review]
LGTM

File: systems/palatine-hill/configuration.nix (hunk 1 above)
Comment (ahuston-0) [reply systems/palatine-hill/configuration.nix:27]: will make the change on my local and get it in another PR. don't want to keep you up to do more approvals

**Pull request merged**